### ISSUE TITLE
Limited Number of Remote Calendars

### ISSUE THREAD

**Issue by basbra** [Ericsson/exchangecalendar#163]
Hi, I experienced a problem after subscribing to 14 calendars overall, 2 of which my own, the other 12 my colleague's calendars. The progress bar stalls, the blue arrow turns and turns and turns... After deleting all but 3, sync works again. It seems this is related to #105 (https://github.com/Ericsson/exchangecalendar/issues/105).

EWS 3.3.2~rc1

Best regards!

**Comment by muthusuba** [comment]
@basbra Did it work fine in 3.3.1, please?

**Comment by basbra** [comment]
@muthusuba No, actually, 3.3.1 has the same issue. Also, the code checked out from the master branch on Monday (April 20) did not work properly (and, btw, offered an "upgrade" to 3.3.2).

**Comment by muthusuba** [comment]
@basbra I guess, we need more info to dig this - assuming this is a problem in 3.3.2 as well :(
screenshot? logs?

**Comment by Saxomania** [comment]
@muthusuba: I have one user with a similar problems, a high number of queue jobs (~150) which will be started again and again as mentioned here (latest post): https://github.com/Ericsson/exchangecalendar/issues/105  -- but no other calendar is configured - only his own personal one

I can provide logs or screenshots. What do you need?

**Comment by bavincen** [comment]
hi @Saxomania 
use this 
https://github.com/ericsson/exchangecalendar/tree/cacheandperformancefix

let us know if queue jobs still high

**Comment by Saxomania** [comment]
hi bavincen,

thanks for reply. The user will be back on friday. Deployed last friday TB 31.7.0 and EWS 3.3.2 maybe this already helped if not i will try the performance fix, i will let you know.

**Comment by bavincen** [comment]
any update?

**Comment by Saxomania** [comment]
Yes, but not very successful.

Yesterday I changed the plugin and saw some changes on the EWS properties but first I had to remove the existing calendar
and create/connect a new one then some more properties were available.

But the main problem still exists, not always but sometimes the queue jobs counts high (peak at 165 jobs running). I cant reproduce
it always but if you create a new meeting/appointment on outlook calendar and then sync on TB calendar sometime the job queue
counts high and sometimes only 10 jobs were in the queue.

Maybe I can provide some logs or something else to check that.

Tomorrow we will create a new TB profile for that user and check it again.

Saxomania

Von: bavincen [mailto:notifications@github.com]
Gesendet: Donnerstag, 11. Juni 2015 13:30
An: Ericsson/exchangecalendar
Cc: Osdoba, Sascha
Betreff: Re: [exchangecalendar] Limited Number of Remote Calendars (#163)


any update?

**Comment by bavincen** [comment]
Ok.. I have an answer.
first of all your queue jobs issue is resolved.
and we have new feature email followup which actually the nummber of updates in mail in total..
no harm :) many of them will be discarded.. you can expect optimized in stable .. thanks

**Comment by basbra** [comment]
Actually, I did not see the problem while running version 3.3.2. After updating to v3.4.0-beta1, the problem was back: One job running an X queued where 1 < X < 12, and no progress from that state :( Finally, a downgrade back to 3.3.2 solved the problem.

**Comment by bavincen** [comment]
Can you please use the latest code. Please refer wiki  to install latest adfon

Thanks

**Comment by Saxomania** [comment]
Dear all,

we create a new TB profile but the problem appears again. Then i tried with the latest beta addon, again - not always but sometimes mostly after doing changes on the calendar with outlook. Sometimes queue jobs 200more. Logfile created. I deleted the informations inside it about server/username but i am not quite sure if deleted all informations so i dont want to upload to the public. Maybe i can get a emailadress to send it directly - not my data but from user to i have be carefully. Logfile is nearly 200mb ;-) High queue job started 16:10 and ended around 16:25!

If you are interested in, give me a sign.

**Comment by muthusuba** [comment]
@Saxomania assuming this is with beta2, please? If so, yes, log files would be good...(better, if you can let us know what exactly is the problem from the log file :) )
Nevertheless, would you be able to zip and share it (via a public dropbox kind of service, if possible), please?

**Comment by bavincen** [comment]
looks like you are modifying recurring event,  there must be huge number of recurrence,
